unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/BaseControllers/SettingsControlController.cs

```diff
@@ -25,6 +25,8 @@ public virtual void Initialize()
 
         public virtual void OnDestroy()
         {
+            if (Settings.i == null) return; // fix for PlayMode tests failing on TearDown
```

Comment:
settings management should be managed by its own controller, this way you could avoid this line